### DISCUSSION TITLE
JsonTable: Feldinhalt ändern

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#2188] [Q&A] [answered]
Hallo,

wie kann ich den Feldinhalt einer Zelle ändern?
Kopfzeile habe ich hinbekommen.
Ich möchte zum Beispiel "open" durch "geöffnet" ersetzen.

Danke

**Comment by mcuiobroker** [comment]
Es in der Doku hierfür nur Hinweise wie man den Wert ändern könnte, kein Beispiel.
Siehe Doku scripts -> jsonTable.

**Comment by AFPass** [comment]
Da stecke ich zu wenig in JS drinnen um noch mit Scripts anzufangen.
Dachte es gibt da so etwas wie bei der Modul Konfiguration und der Field / Label Konfiguration.

**Comment by mcuiobroker** [comment]
Da wüsste ich nicht wie.

**Comment by Zefau** [comment, marked as answer]
In der JsonTable können Werte mittels `format` Parameter geändert werden. In der Konfiguration der Spalten dann für die Spalte `"format": "val => val === 'open' ? 'geöffnet' : val"` reinschreiben.

**Reply by AFPass** [reply]
Danke

**Comment by mcuiobroker** [comment]
Jetzt hab ich es auch gefunden. 
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/module/jsontable/v3.x.x/parameter-format

**Reply by AFPass** [reply]
Danke
